routes/spec.js

```diff
@@ -1740,6 +1746,9 @@ Consider supporting The Slothpixel Project on Patreon to help cover the hosting
                                 },
                               },
                             },
+                            average_skill_level:{
+                              type:'float',
```

Comment:
`float` is not a valid type in OpenAPI spec, you should use `number` instead. Also the missing spaces cause the linting to fail.